### PR TITLE
Physical customer crud

### PR DESCRIPTION
**Pull request by Juan-Pablo-Chavez-AS**

### CODE REVIEW

File: src/controllers/physicalCustomer.controller.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  getPhysicalCustomer,
+  updatePhysicalCustomerRepository
+} from '../repositories/physicalCustomer.repository.js'
+
+export const getPhysicalCustomers = async (req, res) => {
+  try {
+    const physicalCustomerId = req.params.id
+
+    let customers
+    if (physicalCustomerId !== undefined) {
```
Comment (JoseLozadaAD) [inline src/controllers/physicalCustomer.controller.js:14]: Use YODA

File: src/controllers/physicalCustomer.controller.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import {
+  createPhysicalCustomerRepository,
+  deletePhysicalCustomerRepository,
+  getAllPhysicalCustomers,
+  getPhysicalCustomer,
+  updatePhysicalCustomerRepository
+} from '../repositories/physicalCustomer.repository.js'
+
+export const getPhysicalCustomers = async (req, res) => {
```
Comment (JoseLozadaAD) [inline src/controllers/physicalCustomer.controller.js:9]: req, res ? Use better name description. (request, response) sound better

Comment (jacovzap-assuresoft) [reply src/controllers/physicalCustomer.controller.js:9]: Those names are convention of the library of express, please read the documentation https://expressjs.com/es/guide/routing.html

> req, res ? Use better name description. (request, response) sound better

File: src/controllers/physicalCustomer.controller.js (hunk 1 above)
Comment (Juan-Pablo-Chavez-AS) [reply src/controllers/physicalCustomer.controller.js:14]: No need for YODA, physicalCustomerId is a constant. A typo would be caught by the standard library
image

File: src/routes/physicalCustomer.route.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+import {
+  createPhysicalCustomer,
+  deletePhysicalCustomer,
+  getPhysicalCustomers,
+  updatePhysicalCustomer
+} from '../controllers/physicalCustomer.controller.js'
+
+const router = Router()
+
+router.get('/:id?', getPhysicalCustomers)
```
Comment (OmarOporto3) [inline src/routes/physicalCustomer.route.js:12]: Should we have different routes for each (get unique, get all) & specifics imports to have the same agreement :

get an UNIQUE element & another for get ALL elements

Comment (Juan-Pablo-Chavez-AS) [reply src/routes/physicalCustomer.route.js:12]: irrelevant, carry over to next sprint🤙🏻

Comment (OmarOporto3) [reply src/routes/physicalCustomer.route.js:12]: It's OK

**Pull request merged**